.gitignore

```diff
@@ -1,6 +1,8 @@
 .godeps
 vendor
 ./build
+build/
+test
```

Comment:
this will also ignore `internal/test/` which has checked-in source code we don't want ignored